.travis.yml

```diff
@@ -12,10 +12,9 @@ env:
     - PRESET=ARM_REL
     - PRESET=ARM_REL_MIN
 
-    - PRESET=STM32F4
+    - PRESET=STM32FX
```

Comment:
This is better - I wonder if it's true and if this example will work for entire FX family ?